opencadc.gradle

```diff
@@ -18,31 +18,32 @@ checkstyle {
 
 sourceSets {
     test {
-        resources.srcDirs += 'src/test/resources'
+        //resources.srcDirs += 'src/test/resources'
     }
     intTest {
         java {
             compileClasspath += main.output + test.output
-            runtimeClasspath += main.output + test.output
+            //runtimeClasspath += sourceSets.main.runtimeClasspath 
+            runtimeClasspath += compileClasspath
```

Comment:
Is this an optimization?  I can't tell if this is accurate.

Reply:
intTests fail at runtime without this